README.rst

```diff
@@ -43,6 +43,8 @@ Table of Contents
 * `Using the Latest Images`_
 * `Getting Started`_
 * `Usernames and Passwords`_
+* `Enable ADG theme`_
+* `Transifex Translations`_
```

Comment:
- In order to minimize core changes, let's please move these two thing in a separate `.md` document and just add the link at appropriate place here.
- Let's please remove all indentation core changes from this file.

Reply:
done